vulture/core.py

```diff
@@ -74,9 +74,14 @@ def _ignore_class(filename, class_name):
     return _is_test_file(filename) and "Test" in class_name
 
 
-def _ignore_import(_filename, import_name):
+def _ignore_import(filename, import_name):
+    def _is_dunder_init_py(filename):
```

Comment:
I'm rather unhappy with this name, but coulnd't think of a better one. Please feel free to suggest one.

Reply:
Let's just remove the function and use its code directly. Then the two comments can be turned into one method docstring: 

Ignore star-imported names since we can't detect whether they are used. Ignore imports from __init__.py files since they're commonly used to collect objects from a package.